Karamelfile

```diff
@@ -3,6 +3,12 @@ dependencies:
     global:  
       - ndb::mysqld
       - hops::docker_registry
+      - hops::nm
```

Comment:
You don't need the dependency to the node manager.

Reply:
My bad I meant namenode (nn).